### PR TITLE
[4.0] Update Bootstrap to beta-1

### PR DESCRIPTION
**Pull request by C-Lodder**
### WIP

This PR updates Bootstrap to beta-1 which was released this morning. There's still a lot of fixes to be made to cater for ther B/C breaks, so probably not much point in testing.

### Do NOT merge!
--------------------
### To-Do

- [x] Fix grid issue (@ciar4n may need your eyes on this).
- [x] Mobile menu showing on desktop (Possible a responsive utility class update needed).
- [x] Fix toolbar "save" dropdown misalignment
- [x] Investigate font-size mismatch between `alpha-6` and `beta-1`
- [x] Update tests

### CODE REVIEW

File: media/system/scss/_jalert.scss [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+    border-top-color: darken($background, 5%);
+  }
+  .alert-link {
+    color: darken($background, 10%);
+  }
 }
+
+.alert-message {
+  @include alert-variant(theme-color("success"), theme-color("success"), $gray-200);
+}
```
[the remark this answers is not in the document]
Comment (C-Lodder) [reply media/system/scss/_jalert.scss]: woof, done

File: media/system/webcomponents/joomla-tab.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
+(function () {
```
Comment (dgrammatiko) [inline media/system/webcomponents/joomla-tab.js]: @C-Lodder you need to remove the tab files, it's not ready for merging yet

**Comment by C-Lodder** [comment]
It they're being compile but not merged, then they should probably be ignored in thhe Gruntfile.js

File: media/system/webcomponents/joomla-tab.js (hunk 2 above)
Comment (C-Lodder) [reply media/system/webcomponents/joomla-tab.js]: It they're being compile but not merged, then they should probably be ignored in th
he Gruntfile.js

Comment (dgrammatiko) [reply media/system/webcomponents/joomla-tab.js]: Gruntfile is totally stupid (and that's the way it should be), it copies all the files from the dist/js to the media/system/webcomponents.

Comment (C-Lodder) [reply media/system/webcomponents/joomla-tab.js]: Hmm ok. Will revert that file

Comment (dgrammatiko) [reply media/system/webcomponents/joomla-tab.js]: 4 files, thanks

**Comment by brianteeman** [comment]
Please ping when it is ready for testing

**Comment by C-Lodder** [comment]
will do, thanks @brianteeman

**Comment by C-Lodder** [comment]
@brianteeman - apart from the first 2 things on the "to-do" list mentioned in my initial comment, could you give this a once over to see if anything drastic is going wrong, that would be appreciated. FYI you will need to reinstall Joomla to get the background colour variants on the demo modules for Aurora.

**Comment by brianteeman** [comment]
My four hour train journey from an unnamed location in the middle of Wales is almost over so I will have to put this on the todo list for tomorrow am as I suspect I am about to fall asleep in the keyboard

**Comment by brianteeman** [comment]
Some of the fonts seem to be different sizes for example

screenshotr20-35-57
screenshotr20-36-18

**Comment by brianteeman** [comment]
and there is a difference here as well - maybe its padding - not sure only a quick check

screenshotr20-37-46
screenshotr20-38-11

**Comment by brianteeman** [comment]
And another font size difference

screenshotr20-39-56


screenshotr20-39-35

**Comment by brianteeman** [comment]
@C-Lodder login font size and alert font size are now ok thanks - toolbar still the same

**Comment by C-Lodder** [comment]
@brianteeman fixed

**Comment by C-Lodder** [comment]
@wilsonge - This is good to go now. If there are however any glitches I've missed, a new issue can be opened.

**Comment by brianteeman** [comment]
Can this be merged now please?

**Comment by brianteeman** [comment]
Thanks all

**Pull request merged**